env0/resource_project_policy.go

```diff
@@ -165,14 +171,18 @@ func resourcePolicyUpdate(ctx context.Context, d *schema.ResourceData, meta inte
 		payload.MaxTtl = ""
 	}
 
+	if payload.DriftDetectionCron != "" {
+		payload.DriftDetectionEnabled = true
```

Comment:
If drift is set by the schema pass "enabled" = "true" in the payload.

Reply:
Does it support the disable of this config upon attribute removal?

Reply:
golang has default values. Therefore, if the field is removed 'false' will be sent.